### PR TITLE
Create psionic_ritual.txt

### LINKED ISSUE #927: Psionic Ritual

### PR DESCRIPTION
**Pull request by squee1968**
-closes #927

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Psionic Ritual
+ManaCost:4 U U
+Types:Sorcery
+K:Replicate:tapXType<1/Horror>
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Exile | TgtPrompt$ Choose target instant or sorcery card in your graveyard | ValidTgts$ Instant.YouCtrl,Sorcery.YouCtrl | RememberChanged$ True | SubAbility$ DBCopy | SpellDescription$ Exile target instant or sorcery card from a graveyard and copy it. You may cast the copy without paying its mana cost.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt:5]: Missing "Exile CARDNAME."

File: forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+SVar:DBCopy:DB$ Play | Defined$ Remembered | Amount$ All | CopyCard$ True | WithoutManaCost$ True | SubAbility$ ExileSelf
+SVar:ExileSelf:DB$ ChangeZone | Origin$ Stack | Destination$ Exile | Defined$ Self | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckNeeds:Type$Horror
+AI:RemoveDeck:All
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt:10]: Set up appropriate conditions for the AI and see if it plays this card in a semi-logical way.

If it does, then you don't need this tag.

File: forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt (hunk 1 above)
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt:5]: Own is better than Ctrl for the ValidTgts

File: forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Psionic Ritual
+ManaCost:4 U U
+Types:Sorcery
+K:Replicate:tapXType<1/Horror>
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Exile | TgtPrompt$ Choose target instant or sorcery card in your graveyard | ValidTgts$ Instant.YouCtrl,Sorcery.YouCtrl | RememberChanged$ True | SubAbility$ DBCopy | SpellDescription$ Exile target instant or sorcery card from a graveyard and copy it. You may cast the copy without paying its mana cost.
+SVar:DBCopy:DB$ Play | Defined$ Remembered | Amount$ All | CopyCard$ True | WithoutManaCost$ True | SubAbility$ ExileSelf
+SVar:ExileSelf:DB$ ChangeZone | Origin$ Stack | Destination$ Exile | Defined$ Self | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckNeeds:Type$Horror
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt:9]: also add
```
DeckHints:Type$Instant|Sorcery
DeckHas:Ability$Graveyard
```

File: forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt (hunk 2 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt:12]: So the AI didn't play this card at an appropriate time when conditions were good?

Comment (Agetian) [reply forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt:12]: Judging by the looks of the card itself, the AI should manage to play things like that, but might be worth testing again? Maybe something broke at some point or I'm missing something about the card that the AI doesn't handle well :/ At any rate, judging by the card specification alone (Oracle text etc.), I don't see anything that should prevent the AI from playing it.

Comment (squee1968) [reply forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt:12]: Tried this again with several Horrors in play, plenty of instants and sorceries in the yard, more than enough mana, and at a distinct disadvantage on the board, with 1 life left. I don't think conditions could get much worse, and the AI didn't play it. Some of the cards I based this on had this tag, some did not. It's up to you guys, but at least let the card in for now until time permits for AI improvements.

Comment (Agetian) [reply forge-gui/res/cardsfolder/upcoming/psionic_ritual.txt:12]: Alright, I'll try to take a look at the AI part later :)

**Review by Agetian — APPROVED** [review]

**Pull request merged**